src/azure-cli/azure/cli/command_modules/sql/_params.py

```diff
@@ -524,6 +537,15 @@ def load_arguments(self, _):
 
         c.argument('max_size_bytes', help='The new maximum size of the database expressed in bytes.')
 
+        c.argument('read_scale',
+                   arg_type=read_scale_param_type,
+                   help='If enabled, connections that have application intent set to readonly in their connection string may be routed to a readonly secondary replica. '
```

Comment:
I think you don't need to override this in `sql db update` layer if it's already set in `sql db` layer

Reply:
Fixed.